recipes/scikit-umfpack/bld.bat

```diff
@@ -0,0 +1,6 @@
+
+set UMFPACK=%PREFIX%\lib
```

Comment:
So, I know Windows support doesn't exist *yet*. Normally libraries are place in `%LIBRARY_LIB%`, which is just short form for `%PREFIX%\Library\lib`.